cardano-db/src/Cardano/Db/Schema.hs

```diff
@@ -19,6 +19,7 @@ module Cardano.Db.Schema where
 
 import           Cardano.Db.Schema.Orphans ()
 
+import           Cardano.Db.Schema.Types as Types
```

Comment:
My understanding is that the tables required by SMASH have **already** been added and should probably not need to be changed.